django/template/engine.py

```diff
@@ -90,6 +104,15 @@ def template_context_processors(self):
         context_processors += tuple(self.context_processors)
         return tuple(import_string(path) for path in context_processors)
 
+    def get_template_builtins(self, builtins):
+        return [import_library(x) for x in builtins]
+
+    def get_template_libraries(self, libraries):
+        loaded = {}
+        for name, path in libraries.items():
+            loaded[name] = import_library(path)
+        return loaded
+
     @cached_property
```

Comment:
Good catch. This inconsistency isn't necessary.

My reasoning in the description was specific to the ``DjangoTemplates`` backend, since it now runs discovery rather than lazily importing application tag modules when the ``{% load %}`` tag is used.

The only argument I have for using an attribute rather than a cached property is that it's usually better to fail loud and early. Do you remember what the thought was behind using ``cached_property`` for loaders?